src/screens/SUE/SueListScreen.tsx

```diff
@@ -320,7 +319,8 @@ export const SueListScreen = ({ navigation, route }: Props) => {
                       placeholder: texts.sue.filter.alleSortingTypes
                     }
                   ]}
-                  queryVariables={initialQueryVariables}
+                  initialQueryVariables={initialQueryVariables}
+                  initialStartDate={initialStartDate.start_date}
```

Comment:
here `initialStatDate` is an object but in the other component it is a string? i suggest using the same object when naming it the same